tests/importer.rs

```diff
@@ -0,0 +1,252 @@
+use std::collections::{HashMap, HashSet};
+use std::fs;
+
+use hurban_selector::importer;
+use hurban_selector::importer::{Importer, ImporterError};

```

Comment:
Can be written as `use hurban_selector::importer::{self, Importer, ImporterError};`

Reply:
Oh, nice, I didn't know about this syntax! 👍

Reply:
Done.